ci/authn-azure/secrets.yml

```diff
@@ -5,5 +5,3 @@ USER_ASSIGNED_IDENTITY: !var ci/azure/authn-test/user-assigned-id
 USER_ASSIGNED_IDENTITY_CLIENT_ID: !var ci/azure/authn-test/user-assigned-id-client-id
 AZURE_AUTHN_INSTANCE_USERNAME: !var ci/azure/authn-test/instance-user
 AZURE_AUTHN_INSTANCE_PASSWORD: !var ci/azure/authn-test/instance-password
-
```

Comment:
Just to make this clear i would add a commend that SYSTEM_ASSIGNED_IDENTITY is specific variable that taking during Allocate Azure Authenticator Instance.

Reply:
➕ ^

Reply:
Added a note